packages/fast-components-react-base/src/horizontal-overflow/horizontal-overflow.spec.tsx

```diff
@@ -403,5 +419,55 @@ describe("horizontal overflow", (): void => {
             renderedWithImagesAndNextAndPrevious.instance()["handlePreviousClick"]()
         ).toBe(undefined);
     });
+    test("should add resize event listener to the window", (): void => {
```

Comment:
It took me a while to understand what it was these tests were actually doing, I would suggest instead you do:

```ts
test("should add a resize event listener to the window", (): void => {
        let addResizeListenerCount: number = 0;

        /**
         * Mock window.addEventListener to count the number of times
         * a resize event has been added
         */
        window.addEventListener = jest.fn((event: string, callback: any) => {
            if (event === "resize") {
                addResizeListenerCount = addResizeListenerCount ? addResizeListenerCount + 1 : 1;
            }
        });

        expect(addResizeListenerCount).toBe(0);

        const rendered: any = mount(
            <HorizontalOverflow managedClasses={managedClasses}>
                {imageSet1}
            </HorizontalOverflow>
        );

        expect(addResizeListenerCount).toBe(1);
    });
    test("should remove a resize event listener from the window when component unmounts", (): void => {
        let removeResizeListenerCount: number = 0;

        /**
         * Mock window.removeEventListener to count the number of times
         * a resize event has been removed
         */
        window.removeEventListener = jest.fn((event: string, callback: any) => {
            if (event === "resize") {
                removeResizeListenerCount = removeResizeListenerCount ? removeResizeListenerCount + 1 : 1;
            }
        });

        const rendered: any = mount(
            <HorizontalOverflow managedClasses={managedClasses}>
                {imageSet1}
            </HorizontalOverflow>
        );

        expect(removeResizeListenerCount).toBe(0);

        rendered.unmount();

        expect(removeResizeListenerCount).toBe(1);
    });
```

This way you can test that the resize event listener has been added once, and removed once. In this way you are testing that there are 0 resize listeners before mounting, 1 after mounting, similarly the remove event listeners are 0 before mounting, and 1 after unmounting.